From 1be7da8d81bbd52d722ec60250567ea876f6f4b5 Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fabrice.fontaine@orange.com>
Date: Sat, 3 Dec 2016 20:15:41 +0100
Subject: [PATCH] Fix static compilation of json2cbor

json2cbor depends on cjson so tools/json2cbor/json2cbor.o must before
-lcjson

Signed-off-by: Fabrice Fontaine <fabrice.fontaine@orange.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 905fbdf..7b5fefc 100644
--- a/Makefile
+++ b/Makefile
@@ -115,7 +115,7 @@ bin/cbordump: $(CBORDUMP_SOURCES:.c=.o) lib/libtinycbor.a | bin
 	$(CC) -o $@ $(LDFLAGS) $^ $(LDLIBS) -lm
 
 bin/json2cbor: $(JSON2CBOR_SOURCES:.c=.o) lib/libtinycbor.a | bin
-	$(CC) -o $@ $(LDFLAGS) $(LDFLAGS_CJSON) $^ $(LDLIBS) -lm
+	$(CC) -o $@ $(LDFLAGS) $^ $(LDFLAGS_CJSON) $(LDLIBS) -lm
 
 tinycbor.pc: tinycbor.pc.in
 	$(SED) > $@ < $< \
-- 
2.5.0

